packages/edge/flow-typed/npm/eslint-config-prettier_vx.x.x.js

```diff
@@ -0,0 +1,66 @@
+// flow-typed signature: 3b47ab6611b713ec723daf60fcce644f
```

Comment:
Can these files be gitignored?

Reply:
We carry over the format from ACP react native and all the extension are checking in flow-typed. We can raise an issue ticket to look into this later.